mix.exs

```diff
@@ -1,14 +1,14 @@
 defmodule SSHKit.Mixfile do
   use Mix.Project
 
-  @version "0.0.3"
+  @version "0.1.0"
   @source "https://github.com/bitcrowd/sshkit.ex"
 
   def project do
     [app: :sshkit,
      name: "sshkit",
      version: @version,
-     elixir: "~> 1.3",
+     elixir: "~> 1.5",
```

Comment:
You mean a separate PR for the package/`@version`, right?
Not the Elixir version… (because that had to change here).

Again, you're absolutely right. Thanks!